scripts/zuul/script.sh

```diff
@@ -90,7 +90,7 @@ if [ "$T" = "tidy" ]; then
 fi
 
 if [ "$T" = "cmake" ]; then
-  cmake -H. -Bbuild -DCURL_WERROR=ON $C
+  cmake -S. -Bbuild -DCURL_WERROR=ON $C
```

Comment:
The Zuul CI scripts are run on Ubuntu Bionic:

https://github.com/curl/curl/blob/f0f587d3471c8efe8687186e5749a4e88498d53f/zuul.d/jobs.yaml#L36

Bionic only packages CMake 3.10.2: https://packages.ubuntu.com/bionic/cmake

I looked at the list of checks, but I don't see where Zuul CI is. Does it have GitHub Actions integration?

Reply:
Interesting.

If that is true, would not that mean that this didn't work even before this patch (since `-B` and `-S` were both added in cmake v3.13)?

---

> I looked at the list of checks, but I don't see where Zuul CI is. 

They are here: <https://curl.zuul.vexxhost.dev/builds>

> Does it have GitHub Actions integration?

Afaik, it doesn't show up most of the times.

> ![file](https://user-images.githubusercontent.com/20175435/173907886-a56fd166-8abe-4b40-992d-d507c268f795.png)
> https://curl.zuul.vexxhost.dev/status

It looks like the job for this PR is still in queue.